.github/workflows/testing.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Install sycamore
         run: poetry install
       - name: Run tests
-        run: poetry run pytest sycamore/tests/unit/
+        run: poetry run pytest lib/sycamore/sycamore/tests/unit/
```

Comment:
The tests pass, so this must work, but just for my own comprehension I'm trying to make sure I understand the difference between this and something like `cd lib/sycamore && poetry run pytest sycamore/tests/unit/`. If I understand right, this will run in the context of the root pyproject.toml, so it will also have the crawler dependencies, etc?

Reply:
yep, that's right. Will look to see if I can get it to run with just the dataprep lib context

Reply:
I don't necessarily know that that is required. I'm trying to think in terms of virtualenvs. Will I primarily want to operate in a single virtualenv that has everything, or switch virtualenvs based on sub-project. I guess probably the former unless we get incompatible dependencies (which we really should try to avoid).

Reply:
well, on the positive side, I did this and immediately found a bug where a dependency (pypdf2) was added to the root pyproject but wasn't in the lib/sycamore pyproject - and since the lib/sycamore pyproject is what will be used to build the PyPI release, we want to catch that